scripts/labs/argocd/7-kyverno.sh

```diff
@@ -18,16 +18,21 @@ spec:
   source:
     chart: kyverno
     repoURL: https://kyverno.github.io/kyverno
-    targetRevision: 2.6.2
+    targetRevision: 2.7.2
     helm:
       values: |
-        serviceMonitor:
```

Comment:
Why remove the service monitor ? It’s not related to tracing

Reply:
I put that in the PR.  With that, not all of the span names rendered with tempo . Try it and without.

Reply:
Can you elaborate ? What spans are missing ?